CarSharingApp.PublicApi/appsettings.json

```diff
@@ -5,5 +5,42 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+
+  "KeyVaultConfig": {
+    "KVUrl": "https://glebcarsharingakeyvault.vault.azure.net/",
+    "TenantId": "55f04023-a02c-4509-a195-0f519de89e74",
+    "ClientId": "f90c5d2a-e525-444d-af16-82c319cbf3ad",
+    "ClientSecretId": "o5X8Q~a.vEube6KcNN0xHagMCrRN29SJ64sbbblA"
+  },
+
+  "ConnectionStrings": {
+    "MongoDbAtlasCloud": "",
+    "MongoDbLocal": "mongodb://127.0.0.1:27017",
```

Comment:
better to has one connection string setting depends on Environment.

please use appsetting.Development.json for dev (local), appsettings.json for PRODUCTION.